Makefile

```diff
@@ -366,6 +387,10 @@ loop: mod core
 	@$(CC) -c -o $(PREFIX)/loop_ping.o test/loop_ping.c $(CFLAGS) -ggdb
 
 units: mod core lib $(UNITS_OBJ) $(MOD_OBJ)
+	@if [ $(UNITS) -eq 0 ]; then \
```

Comment:
I'm no Makefile expert...

What if you add a 'global' variable on top:

UNITS?=0 #by default

In the units, make a phony target that sets UNITS=1 at the start of the dependencies.

Does that make sense? Or am I hoping for something?